### PR TITLE
Exposing defaultFormatter to allow logging format customisation

### PR DESCRIPTION
**Pull request by suls**
Hi @kevinoliver and @mosesn - and sorry for the recent spam ..

The change below would drastically reduce the boilerplate needed to change logging format using the current API; see my example here (https://github.com/suls/custom-log-for-twitter-server/commit/da540a21d45c9a4d7f07708672535c89daf760a2)

Please let me know your thoughts and any feedback of course ! It's a bit late for me right now but I am happy to change the commit message later on if that is needed :v: 

Cheers and thanks for your time!

### CODE REVIEW

File: util-logging/src/main/scala/com/twitter/logging/App.scala [hunk 1]
```diff
@@ -58,14 +60,15 @@ trait Logging { self: App =>
...
+        ConsoleHandler(level = level, formatter = defaultFormatter)
       else
         FileHandler(
           output,
           rollPolicyFlag(),
           appendFlag(),
           rotateCountFlag(),
-          level = level
+          level = level,
+          formatter = defaultFormatter
```
Comment (kevinoliver) [inline util-logging/src/main/scala/com/twitter/logging/App.scala]: mind keeping the ordering the same as the method's? in other words, just move formatter right before level.

**Comment by kevinoliver** [comment]
this looks great, thanks.

mind just polishing up the commit message a bit and fixing that one nitpick and then i'll merge this in.

**Comment by mosesn** [comment]
LGTM, :+1: to @kevinoliver's suggestions

**Comment by suls** [comment]
Great! Will do both! And thanks again guys!

**Comment by suls** [comment]
- [x] improve commit message
- [x] reorder parameters (not only `FileHandler` but also `ConsoleHandler`)
- [x] val +1, var -1 :v: 

I hope the last point doesn't turn this PR from _almost accepted_ to _not on my watch_ .. I just couldn't resist :tada:

**Comment by mosesn** [comment]
LGTM

**Comment by kevinoliver** [comment]
pulling this in locally, thanks!

**Comment by kevinoliver** [comment]
@suls cool, just got merged this merged in locally. Should show up on the `develop` branch this Monday. Thanks again.

**Pull request closed without merging**